cmd/contour/serve.go

```diff
@@ -46,6 +46,7 @@ import (
 	xdscache_v3 "github.com/projectcontour/contour/internal/xdscache/v3"
 	"github.com/projectcontour/contour/pkg/config"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/collectors"
```

Comment:
Did updating controller-runtime also update prometheus versions?

Reply:
It did, yeah

Reply:
FWIW, the code change is because the functions in the `prometheus` package are marked deprecated and generated a linter error, however the `collector` implementations just call back into the `prometheus` package for now, so no behavior change: https://github.com/prometheus/client_golang/blob/master/prometheus/collectors/process_collector.go#L49-L55